### PR TITLE
fix: validation added for the value; it was not clear for the end-user

### PR DESCRIPTION
**Pull request by UtmostCreator**

### CODE REVIEW

File: controllers/CalendarController.php [hunk 1]
```diff
@@ -104,7 +104,7 @@ public function actionEdit($id = null)
         $model = ($id) ? $this->findModel($id) : new ExternalCalendar($this->contentContainer);
 
         try {
-            if ($model->load(Yii::$app->request->post())) {
+            if ($model->load(Yii::$app->request->post()) && $model->validate()) {
                 (new ICalSync(['calendarModel' => $model, 'skipEvents' => true]))->syncICal();
                 if(!$model->hasErrors()) {
```
Comment (buddh4) [inline controllers/CalendarController.php:109]: Shouldn't this already be covered by this line?

Comment (MasterMindNET) [reply controllers/CalendarController.php:109]: AFAIK
load only pupolates the attributes, no validation is performed on it, you can try to test it out, in order to see difference.
it contines the execution only after all the $_POST datat is valid.
`$model->load(\Yii::$app->request->post()); // $model->attributes = \Yii::$app->request->post('ContactForm');`
[Validating Input](https://www.yiiframework.com/doc/guide/2.0/en/input-validation)

Comment (buddh4) [reply controllers/CalendarController.php:109]: You are right, I missread the line.

File: views/calendar/_form.php [hunk 2]
```diff
@@ -4,23 +4,26 @@
...
     $model->color = $contentContainer->color;
-} elseif ($model->color == null) {
+} elseif (!isset($model->color)) {
     $model->color = '#d1d1d1';
 }
 
 \humhub\modules\external_calendar\assets\Assets::register($this);
 ?>
 <div class="calendar-extension-calendar-form">
+    <?php Pjax::begin(); ?>
```
Comment (buddh4) [inline views/calendar/_form.php:23]: Why the Pjax?

Comment (MasterMindNET) [reply views/calendar/_form.php]: I have removed it. I  beilive it is not used there.  I have merged with latest master branch, so please check if it is okay, thank you!

**Comment by luke-** [comment]
@yurabakhtin Can you please review? Is this still required?

**Review by yurabakhtin — APPROVED** [review]

**Comment by yurabakhtin** [comment]
@luke- 

> Can you please review? Is this still required?

Yes, I think this PR should be merged because it adds a missed validation.

**Pull request merged**